ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/Coverage/Clusters/ClusterDetailsTable.tsx

```diff
@@ -132,26 +143,30 @@ function ClusterDetailsTable({
     }
 
     const renderTableContent = () => {
-        return scanResults?.checkResults.map(({ checkName, rationale, clusters }) => {
-            const scanConfigUrl = generatePath(complianceEnhancedScanConfigDetailPath, {
-                scanConfigId: checkName,
-            });
-
+        return scanResults?.checkResults.map((checkResult) => {
+            const { checkName, rationale, clusters } = checkResult;
             const status = getStatusByClusterId(clusters);
             const statusObj = status ? getClusterResultsStatusObject(status) : null;
 
             return (
                 <Tr key={checkName}>
                     <Td modifier="truncate">
                         <>
```

Comment:
Not sure why I added that fragment. I was messing around with this section a bit to try to get the alignment right so that's probably how it ended up in there. I'll remove it